os/kernel/binary_manager/binary_manager_load.c

```diff
@@ -224,14 +224,17 @@ static int binary_manager_load(int bin_idx)
 			{
 				ret = binary_manager_read_header(BINARY_USERAPP, devpath, &user_header_data, true);
 			}
-			if (ret != OK) {
+			if (ret == BINMGR_OK) {
+				printf("%s Header Checking Success\n", BIN_NAME(bin_idx));
+			} else {
+				printf("Invalid Header data, name : %s, devpath : %p\n", BIN_NAME(bin_idx), devpath);
 				if (--bin_count > 0) {
-					bmdbg("Failed to read header %s, try to read binary in another partition\n", devpath);
 					BIN_USEIDX(bin_idx) ^= 1;
+					printf("Try to read another partition %s\n", GET_PARTNAME(BIN_USEIDX(bin_idx)));
```

Comment:
How about adding tab like below?
```
printf("\tTry to
```
Then it is much better to understand that this is because of invalid header data.

Reply:
I added prefix instead of tab.